tasks/tests/test_prime_workflow.py

```diff
@@ -0,0 +1,218 @@
+# -*- coding: utf-8 -*-
+from locust.test.testcases import LocustTestCase
+
+from tasks import WorkflowTasks
+from locustfiles.workflow import WorkflowUser
+from utils.constants import (
+    MOVE_TASK_ORDER,
+    MTO_SHIPMENT,
+    MTO_SERVICE_ITEM,
+    PAYMENT_REQUEST,
+)
+from utils.base import ImplementationError
+
+
+class TestWorkflowTasks(LocustTestCase):
+    def setUp(self):
+        super().setUp()
+
+        class TestWorkflowUser(WorkflowUser):
+            host = "127.0.0.1"
+
+        self.locust = TestWorkflowUser(self.environment)
+        self.taskset = WorkflowTasks(self.locust)
+
+    def test_add_stored_mto(self):
+        # Under test: add_stored function
+        # Mocked: Data objects are simplified versions
+
+        # Scenario: We add a move, and then another move
+        # Expectation: It should store the move into current move and then replace it, since we only keep one move.
```

Comment:
These comments should be in the docstring for this function, but that's a really minor nitpick.

Reply:
Put the Under test and Mocked comments into docstrings. The Scenario and Expectation are regular comments.